src/network/core_utils.rs

```diff
@@ -22,6 +22,10 @@ use sysctl::{Sysctl, SysctlError};
 
 use super::netlink;
 
+pub const IPVLAN_MODE_L2: u16 = 0;
+pub const IPVLAN_MODE_L3: u16 = 1;
+pub const IPVLAN_MODE_L3S: u16 = 2;
+
```

Comment:
are there no const in the upstream library somewhere?

Reply:
unfortunately these are missing.